Week3/homework/2-step3.js

```diff
@@ -1,8 +1,28 @@
-'use strict';
+"use strict";
+function sayThree(x) {
+  console.log(x + " is divisible by 3 ");
+}
+function sayFive(x) {
+  console.log(x + " is divisible by 5 ");
+}
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  for (let i = startIndex; i <= stopIndex; i++) {
+    values.push(i);
+  }
+  console.log(values);
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] % 3 === 0 && values[i] % 5=== 0 ) {
```

Comment:
It is unnecessary to test for the combination of divisibility by 3 and 5 at the same time. This is handled automatically when you you test for divisibility by 3 and by 5 separately:

```js
if (values[i] % 3 === 0) {
  threeCallback(values[i]);
}
if (values[i] % 5 === 0) {
  fiveCallback(values[i]);
}
```